Core/Relationships/RelationshipResolver.cs

```diff
@@ -216,15 +219,13 @@ public void AddModulesToInstall(IEnumerable<CkanModule> modules)
                     }
                     else
                     {
-                        throw new InconsistentKraken(string.Format("{0} conflicts with {1}, can't install both.", module,
-                            listed_mod));
+                        inconsistencies.Add(string.Format("{0} conflicts with {1}, can't install both.", module, listed_mod));
                     }
-                }
-
+                }                    
                 user_requested_mods.Add(module);
                 Add(module, new SelectionReason.UserRequested());
```

Comment:
The user may want to deselect the other mod instead. As long as we're telling them what the problem is, flagging the conflicting mods, and disabling the Apply button, I think this is consistent with past behaviour and makes sense. The user has selected this mod because they want it. At this stage, we don't know which of the conflicts they prefer, so better to accept the selection and let them fix it. Otherwise, they may need to scroll to the other mod, unselect it, then scroll back to here and re-select this mod, making far more effort.